framework/src/modules/chain/submodules/blocks/utils.js

```diff
@@ -625,52 +625,53 @@ class Utils {
 	// eslint-disable-next-line class-methods-use-this
 	aggregateBlocksReward(filter, cb) {
 		const params = {};
+		const filters = { address: filter.address };
+		const options = { extended: true };
+
+		library.storage.entities.Account.get(filters, options)
+			.then(accounts => {
+				const account = accounts[0];
+				if (!account) {
+					return setImmediate(cb, 'Account not found');
+				}
 
-		library.logic.account.get({ address: filter.address }, (err, account) => {
-			if (err) {
-				return setImmediate(cb, err);
-			}
-
-			if (!account) {
-				return setImmediate(cb, 'Account not found');
-			}
-
-			params.generatorPublicKey = account.publicKey;
-			params.delegates = ACTIVE_DELEGATES;
+				params.generatorPublicKey = account.publicKey;
+				params.delegates = ACTIVE_DELEGATES;
 
-			if (filter.start !== undefined) {
-				params.fromTimestamp = Math.floor(
-					(filter.start - new Date(EPOCH_TIME).getTime()) / 1000
-				);
-				params.fromTimestamp = params.fromTimestamp.toFixed();
-			}
+				if (filter.start !== undefined) {
```

Comment:
I know it was like this before but maybe `!filter.start` would be a better check, because it would work if filter.start is null :D Same goes below.